packages/client/src/components/ProductList/ProductList.component.js

```diff
@@ -22,6 +23,37 @@ export default function ProductList() {
     fetchProducts();
   }, []);
 
+  const fetchDropdown = (categoryId) => {
+    fetch(`${apiURL()}/products?category=${categoryId}`)
+      .then((res) => res.json())
+      .then((item) => {
+        setProducts(item);
+      });
+  };
+  useEffect(() => {
+    fetchDropdown(category);
+  }, [category]);
```

Comment:
Again, this should probably be the responsibility of the container, same for the sorting. 

The component should inform the container of what should happen, the container knows how to do it.

Reply:
I have moved fetch function and logic to collection page in container.